helm-org.el

```diff
@@ -58,9 +58,11 @@ NOTE: This will be slow on large org buffers."
   (org-show-entry))
 
 (cl-defun helm-source-org-headings-for-files (filenames
-                                              &optional (min-depth 1) (max-depth 8))
+                                              &optional (min-depth 1) (max-depth 8) (parents nil))
   (helm-build-sync-source "Org Headings"
-    :candidates (helm-org-get-candidates filenames min-depth max-depth)
+    :candidates (helm-org-get-candidates filenames min-depth max-depth parents)
+    :candidate-transformer (if parents 'reverse nil)
+    :after-init-hook 'helm-follow-mode
```

Comment:
> Did you try using :follow 1 instead of :after-init-hook ?

Yes, it does activate follow-mode, but it happens before the preselection, causing the point-jumping behavior.

>  But anyway, I am not sure it is a good idea to enable follow-mode by default (not what I want for myself anyway).

I imagine that it's not for everyone, no.  But for my use case, it definitely helps.  For example, when I would use this command, I'm buried somewhere deep in a 12-level outline, so having the buffer follow to each heading as I select it lets me see the context around it, and then jump back to where I was if I `C-g` out of it.  It's a no-harm-done operation that can be very helpful, so that's why I would like to activate it for this command.  But if you think it shouldn't be, it could be left off.  I would like to find a way to offer support for it, though.  :)

> Also I remember now I have created helm-after-preselection-hook some days ago for this same reason, but still using :follow is better IMO.

I noticed that hook, but from my limited understanding, it appears that it's a global hook that would apply to all Helm sources--and definitely not all Helm sources should have follow-mode activated.  Is there a way to use that hook for individual sources?

Thanks.